test/tests/main.js

```diff
@@ -1,10 +1,36 @@
 const expect = chai.expect;
 describe('main output', function () {
+  it('can initiate main class', function () {
+    expect(p5.soundOut.input).to.have.property('gain');
+    expect(p5.soundOut.input).to.have.property('context');
+    expect(p5.soundOut.output).to.have.property('gain');
+    expect(p5.soundOut.output).to.have.property('context');
+    expect(p5.soundOut.limiter.threshold.value).to.equal(-3);
+    expect(p5.soundOut.limiter.ratio.value).to.equal(20);
+    expect(p5.soundOut.limiter.knee.value).to.equal(1);
+    expect(p5.soundOut.audiocontext).to.have.property('audioWorklet');
+    expect(p5.soundOut.audiocontext).to.have.property('baseLatency');
+    expect(p5.soundOut.meter).to.have.property('gain');
+    expect(p5.soundOut.meter).to.have.property('context');
+    expect(p5.soundOut.fftMeter).to.have.property('gain');
+    expect(p5.soundOut.fftMeter).to.have.property('context');
+    expect(p5.soundOut.soundArray).to.be.an('array');
+    expect(p5.soundOut.parts).to.be.an('array');
+    expect(p5.soundOut.extensions).to.be.an('array');
+
+    expect(p5.soundOut._silentNode).to.have.property('gain');
+    expect(p5.soundOut._silentNode).to.have.property('context');
+    expect(p5.soundOut._silentNode.gain.value).to.equal(0);
+
+    console.log(p5.soundOut);
```

Comment:
```suggestion
```

some repos have a linting rule to remove console.logs which I always find helpful to clean up my own 😄